src/test/scala/no/ndla/audioapi/controller/HealthControllerTest.scala

```diff
@@ -10,16 +10,128 @@ package no.ndla.audioapi.controller
 
 import no.ndla.audioapi.{TestEnvironment, UnitSuite}
 import org.scalatra.test.scalatest.ScalatraFunSuite
+import org.mockito.Mockito._
+import scalaj.http.HttpResponse
 
 class HealthControllerTest extends UnitSuite with TestEnvironment with ScalatraFunSuite {
 
-  lazy val controller = new HealthController
+  val audioSearchBody = s"""{
```

Comment:
Lurer på om det ikke er bedre å gjøre om denne til en vanlig `SearchResult` case class. Og i stedet bruke Json4S sin `write` -funksjon for å gjøre om til json de stedene vi trenger det. Lettere å vedlikeholde `case class`er enn tekststrenger som skal matche case class.

Gjelder forøvrig de andre tilfellene under også